Unable to build 1.0, bson.h not found [FreeBSD 12.1]

I saw the release of 1.0, so I cloned the repository into a new directory and started building.

The first run of `./autogen.sh` complained about libbson, so the package was installed. The second run completed without error. There is no error reported by `./configure` either.

`make` fails with:
```
 FvwmMFL.c:30:10: fatal error: 'bson/bson.h' file not found
 #include <bson/bson.h>
```
Build system is FreeBSD 12.1-RELEASE-p5 with all packages up-to-date as of 2020-09-03.

[configure.log](https://github.com/fvwmorg/fvwm3/files/5171697/configure.log)
[make.log](https://github.com/fvwmorg/fvwm3/files/5171699/make.log)

--- a/modules/FvwmMFL/FvwmMFL.c
+++ b/modules/FvwmMFL/FvwmMFL.c
@@ -27,7 +27,11 @@
 #include <errno.h>
 #include <stdbool.h>
 
+#if defined(__FreeBSD__)
+#include <libbson-1.0/bson.h>
+#else
 #include <bson/bson.h>
+#endif
 
 #include <event2/event.h>
 /* FIXME: including event_struct.h won't be binary comaptible with future

